tests/unit/test_configurator/test_configurator_metrics_exporter.py

```diff
@@ -36,11 +36,30 @@ def test_configure_metrics_exporter_disabled(
         trace_mocks.get_tracer_provider().get_tracer.assert_not_called()
         mock_meterprovider.assert_not_called()
 
+    def test_configure_metrics_exporter_sw_enabled_false(
+        self,
+        mocker,
+        mock_apmconfig_metrics_enabled_false,
+        mock_pemreader,
+        mock_meterprovider,
+    ):
+        # Mock Otel
+        trace_mocks = get_trace_mocks(mocker)
+
+        # Test!
+        test_configurator = configurator.SolarWindsConfigurator()
+        test_configurator._configure_metrics_exporter(
+            mock_apmconfig_metrics_enabled_false,
+        )
+        mock_pemreader.assert_not_called()
+        trace_mocks.get_tracer_provider.assert_not_called()
+        trace_mocks.get_tracer_provider().get_tracer.assert_not_called()
+        mock_meterprovider.assert_not_called()
 
-    def test_configure_metrics_exporter_flag_not_set(
```

Comment:
This must have tested an old flag that was temporarily implemented during early Lambda development. At some point I removed the flag but not the test, which didn't seem to be testing anything meaningful anymore. So I've repurposed to test new metrics enabled config.